pages/11ty/collections.filter.js

```diff
@@ -0,0 +1,22 @@
+/** get value from object by path */
+const get = (item, path) => path.split('.').reduce((acc, key) => acc && acc[key], item);
+
+/** filter to include only items that have a property with passed value */
+function filter(collection, prop, value) {
+  const matcher = arguments.length === 2 ?
```

Comment:
I'm not sure about the proper understanding of njk, but it seems strange to me.
Do we really have to check the number of arguments passed? And what if there is only one?

Reply:
That's the way to strictly separate two usecases of the filter:
1) We do not pass the value (third argument in definition, second in filter usage in njk) 
  - `collection | filter('visible')` -  - so we are searching for items with truthy 'visible' field
  - Same with `collection | exclude('visible')` - we are searching for items with falsy 'visible' field
2) We pass the value to the filter:
 - `collection | filter('visible', true)` -  we are searching for items with exact `true` boolean value
 - `collection | exclude('order', 0)` - we are searching for items with order field not equal to 0 (so false, NaN, etc - are acceptable)

Reply:
Update the logic to look more in a ES6 style, clarify documentation